### PR TITLE
docs: remove troubleshooting guide

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: docs/src/intro-js.md [hunk 1]
```diff
@@ -102,6 +102,13 @@ npx playwright show-report
 
 <img width="1392" alt="HTML Reporter" src="https://user-images.githubusercontent.com/13063165/212743312-edf1e8ed-3fc2-48aa-9c93-24ae3e36504d.png" />
 
+## System requirements
+
+- Node.js 16 or Node.js 18.
```
Comment (mxschmitt) [inline docs/src/intro-js.md:107]:
```suggestion
- Node.js 16+
```
maybe that? What about Node.js 20?

File: docs/src/test-typescript-js.md [hunk 2]
```diff
@@ -87,3 +87,16 @@ In `package.json`, add two scripts:
 The `pretest` script runs typescript on the tests. `test` will run the tests that have been generated to the `tests-out` directory. The `-c` argument configures the test runner to look for tests inside the `tests-out` directory.
 
 Then `npm run test` will build the tests and run them.
+
+## Transpilation issues
+
+Using dynamic imports inside a function passed to various `evaluate()` methods is not supported. This is because Playwright uses `Function.prototype.toString()` to serialize functions, and transpiler will sometimes replace dynamic imports with `require()` calls, which are not valid inside the web page.
```
Comment (mxschmitt) [inline docs/src/test-typescript-js.md:93]: imo this fits better in evaluating.md since its not test-runner specific.

**Review by mxschmitt — APPROVED** [review]

File: docs/src/intro-js.md (hunk 1 above)
Comment (dgozman) [reply docs/src/intro-js.md:107]: Last time I checked we didn't support Node 20 😄  Do we now?

Comment (mxschmitt) [reply docs/src/intro-js.md:107]: We do!

**Pull request merged**